docs/topics/item-pipeline.rst

```diff
@@ -112,7 +119,7 @@ format::
            self.file.close()
 
        def process_item(self, item, spider):
-           line = json.dumps(dict(item)) + "\n"
+           line = json.dumps(dict(ItemAdapter(item).items())) + "\n"
```

Comment:
is .items() required here?

Reply:
[It is not](https://github.com/scrapy/itemadapter/blob/v0.0.6/tests/test_adapter.py#L149-L153), thanks for noticing.